src/tapis-hooks/jobs/useSubmit.ts

```diff
@@ -0,0 +1,46 @@
+import { useMutation } from 'react-query';
+import { Jobs } from '@tapis/tapis-typescript';
+import { submit } from 'tapis-api/jobs';
+import { useTapisConfig } from 'tapis-hooks';
+
+type SubmitHookParams = {
+  request: Jobs.ReqSubmitJob,
+  onSuccess?: (data: Jobs.RespSubmitJob) => any,
+  onError?: (error: any) => any
+}
+
+const useSubmit = () => {
+  const { basePath, accessToken } = useTapisConfig();
+  const jwt = accessToken?.access_token || '';
+
+  // The useMutation react-query hook is used to call operations that make server-side changes
+  // (Other hooks would be used for data retrieval)
+  //
+  // In this case, loginHelper is called to perform the operation, with an onSuccess callback
```

Comment:
The comments here should be changed to reference job submission instead of login.